src/general/OptionsMenu.cs

```diff
@@ -636,6 +653,40 @@ private void DisplayResolution()
             .FormatSafe(screenResolution.x, screenResolution.y);
     }
 
+    /// <summary>
+    /// Displays the GPU name, the display driver name and used video memory
+    /// </summary>
+    private void DisplayGpuInfo()
+    {
+        var videoAdapterName = VisualServer.GetVideoAdapterName();
+        gpuName.Text = TranslationServer.Translate("AUTO_GPU_NAME")
+            .FormatSafe(videoAdapterName);
+
+        if (OS.GetCurrentVideoDriver() == OS.VideoDriver.Gles2)
+        {
+            // Gles2 is being used
+            displayDriverName.Text = TranslationServer.Translate("AUTO_DISPLAY_DRIVER_NAME")
+                .FormatSafe(TranslationServer.Translate("GLES2"));
+        }
+        else
+        {
+            // Gles3 is being used
+            displayDriverName.Text = TranslationServer.Translate("AUTO_DISPLAY_DRIVER_NAME")
+                .FormatSafe(TranslationServer.Translate("GLES3"));
+        }
+
+        float videoMemoryInMebibytes = VisualServer.GetRenderInfo(VisualServer.RenderInfo.VideoMemUsed);
+
+        // Convert to mebibytes
+        videoMemoryInMebibytes /= Constants.MEBIBYTE;
+
+        // Round to 2 places after the floating point
+        decimal decimalVideoMemory = Math.Round((decimal)videoMemoryInMebibytes, 2);
```

Comment:
Probably just use `Mathf.Round` here or cast to float, `decimal` type is used nowhere else in the game so I would not use it here for code consistency's sake.

Reply:
I did `Mathf.Floor(videoMemoryInMebibytes * 100) / 100;` to keep the two digits after the comma, which avoids the decimal type, though that approach seems a bit dirty. That, and I realized that always rounding down would be somewhat more consistent behavior, though that probably wouldn't matter that much here.

Reply:
I'm pretty sure `Mathf.Round` exists... what's the problem with using that?

Reply:
I thought that rounding up sometimes (lying about used memory) and down other times (lying about not using memory) would have been a problem but on second thought, 0.01 mebibytes dont matter that much. I'll use `Mathf.Round()`.